tests/test_integration.py

```diff
@@ -202,12 +202,6 @@ def test_noise_model_qasm_simulator(self, monkeypatch):
             dev = qml.device("qiskit.aer", wires=2, noise_model="test value")
         assert cache[-1] == {"noise_model": "test value"}
 
-    def test_invalid_noise_model(self):
```

Comment:
Why are we removing this? Should we add any alternative tests for this?

Reply:
As far as I can see, it just tests that Qiskit raises an error if the noise model is invalid, it doesn't test behaviour on our end. I don't see why we would do that, and I don't see a reason to maintain tests that we have to update if Qiskit changes an error message.

Reply:
So I can update it to reflect what Qiskit does now, but personally I'd rather just remove it.